project/CelebornBuild.scala

```diff
@@ -389,6 +391,17 @@ object Utils {
     require(ALL_SCALA_VERSIONS.contains(v), s"found not allow scala version: $v")
     v
   }
+
+  def removeDependenciesTransformer: xml.Node => xml.Node = { node =>
```

Comment:
add a comment to explain why we are excluding all dependencies.

Reply:
@cfmcgrady done